packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -105,6 +107,7 @@
                   <include>io.prometheus:simpleclient_common</include>
                   <include>com.yammer.metrics:metrics-core</include>
                   <include>org.apache.spark:spark-streaming-kafka-0-10_${scala.binary.version}</include>
+                  <include>org.apache.spark:spark-token-provider-kafka-0-10_${scala.binary.version}</include>
```

Comment:
Seems like its possible that `scala.binary.version` is `2.11` when compiling by default and it can conflict here because spark 3 only uses `2.12` ? We should probably override scala versions as well by default with the spark3 maven profile so that such scenarios do not happen.

Reply:
okay I'll override scala version inside ```hudi-spark3```